spec/quiet_quality/cli/arg_parser_spec.rb

```diff
@@ -137,6 +139,15 @@ def self.expect_usage_error(desc, arguments, matcher)
       expect_usage_error("-Afoo_bar", ["-Afoo_bar"], /Unrecognized annotator/i)
     end
 
+    describe "logging options" do
```

Comment:
Maybe worth a test describing the `global: {logging: nil}` case? (We may want to rename that to its own logging level later, like :normal or :standard. But that probably belongs in #42)